store/file/q4.go

```diff
@@ -141,30 +119,6 @@ func (q4 *Q4) AxisHalf(ctx context.Context, axisType rsmt2d.Axis, axisIdx int) (
 	}, nil
 }
 
-func (q4 *Q4) Size(context.Context) int {
+func (q4 *q4) Size(context.Context) int {
```

Comment:
Just let it access directly from header method

Reply:
Actually, that size check isn't really needed there, as it happens on ODSQ4 and Q4 isnt general anymore